### DISCUSSION TITLE
503 Ergast

### DISCUSSION THREAD

**Discussion by McHouse** [theOehrly/Fast-F1#556] [Q&A] [answered]
What happens with Ergast??
From 00h approx. until now 18:30 CET today 22/Mar/24, Ergast is at err 503.
Can anyone shed some light?
Thanks in advance.

**Comment by theOehrly** [comment, marked as answer]
The Ergast server is not running stable. It has a lot of downtime at the moment. But that likely won't be fixed anymore. I don't have any influence over that. 
The whole infrastructure will be completely replaced for the next season. But until then, we probably just have to deal with it.

**Comment by khulenikhil** [comment]
Hi guys, just an update. The API is back online.

[https://ergast.com/mrd/](https://ergast.com/mrd/)

**Reply by Alia1999** [reply]
Hi @khulenikhil is ergast currently working. I'm getting errors just trying to import it but I'm not sure what the issue is 

```
from fastf1.ergast import Ergast
ImportError: cannot import name 'Ergast' from 'fastf1.ergast' (C:\Users\alia\AppData\Local\Programs\Python\Python39\lib\site-packages\fastf1\ergast.py)
```

**Reply by theOehrly** [reply]
Import errors are not caused by Ergast itself. I think you probably just have an old version of FastF1 installed. Please use ``pip install --upgrade fastf1`` to update to the newest version.
If this doesn't solve the problem, please start a separate discussion as your problem is unrelated to the problem that was discussed here.
